canmatrix/canmatrix.py

```diff
@@ -170,6 +175,31 @@ def normalizeValueTable(table):
     return {int(k): v for k, v in table.items()}
 
 
+class SignalValue:
+    """Proxy class to hold a Signal and value
+
+    Not using class "SignalValue(object):" on purpose to proxy magic methods
```

Comment:
Are you saying you can't inherit from `object` (in other words, you have to use old style classes in py2) to specify your own `__getattr__`?  If so, [my quick check](https://repl.it/MExN) seems to work fine.

Reply:
`__getattr__` works bu not for __eq__, __lt__, etc.

Reply:
I was thinking on my lunch walk and agree that my example is irrelevant.  I'll try again.  But, if this truly only can be done on old-style classes how would it work in Python 3?

Reply:
Alrighty, you shifted away from that, but, FYI, perhaps [`__getattribute__()`](https://docs.python.org/2/reference/datamodel.html#object.__getattribute__) was what you were missing?

Reply:
Same issue with `__getattribute__` see the note in documentation.